### ISSUE TITLE
Image attribute not returning correct url for some urls.

### ISSUE THREAD

**Issue by parthdesai93** [microlinkhq/metascraper#94]
### Prerequisites

- [x] I'm using the last version.
- [x] My node version is the same as declared as `package.json`.

### Subject of the issue

Scraper returns invalid image url for [this article](https://www.thestar.com/news/canada/2018/06/05/rob-ford-estate-file-shows-11-million-in-assets-and-no-indication-of-any-payout-to-widow-and-children.html).

The image url is supposed to be: 
```https://images.thestar.com/9-uSmF6Dlq_9CXyobH3WL_dQmrs=/1200x799/smart/filters:cb(1528308000000)/https://www.thestar.com/content/dam/thestar/news/canada/2018/06/05/rob-ford-estate-file-shows-11-million-in-assets-and-no-indication-of-any-payout-to-widow-and-children/ford_funeral.jpg```

Actual image url being returned is 
```https://images.thestar.com/9-uSmF6Dlq_9CXyobH3WL_dQmrs=/1200x799/smart/filters:cb(1528308000000)/https:/www.thestar.com/content/dam/thestar/news/canada/2018/06/05/rob-ford-estate-file-shows-11-million-in-assets-and-no-indication-of-any-payout-to-widow-and-children/ford_funeral.jpg```

Notice the missing ```/``` in ```https:/www.thestar.com....```

Metascraper is using the package ```normalize-url``` which strips the slash. 
### Steps to reproduce
```js
let url = 'https://www.thestar.com/news/canada/2018/06/05/rob-ford-estate-file-shows-11-million-in-assets-and-no-indication-of-any-payout-to-widow-and-children.html';

const html = await makeRequest(url) // makeRequest function just uses request-promise-native and returns html. 
const metadata = await metascraper({html, url});

return metadata;
```

### Expected behaviour

```metadata.image``` should be ```https://images.thestar.com/wL8M2wl0ydDGkwl-9HRjo81Kwww=/1200x799/smart/filters:cb(1528311600000)/https://www.thestar.com/content/dam/thestar/news/canada/2018/06/05/rob-ford-estate-file-shows-11-million-in-assets-and-no-indication-of-any-payout-to-widow-and-children/ford_funeral.jpg```

### Actual behaviour
```metadata.image``` is ```https://images.thestar.com/wL8M2wl0ydDGkwl-9HRjo81Kwww=/1200x799/smart/filters:cb(1528311600000)/https:/www.thestar.com/content/dam/thestar/news/canada/2018/06/05/rob-ford-estate-file-shows-11-million-in-assets-and-no-indication-of-any-payout-to-widow-and-children/ford_funeral.jpg```

**Comment by Kikobeats** [comment]
Thanks for reporting, I opened an issue at normalized-url repository

https://github.com/sindresorhus/normalize-url/issues/65

**Comment by parthdesai93** [comment]
Thanks :)

**Comment by Kikobeats** [comment]
Released at `3.11.8` 🥇
